<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFlipWidget</class>
 <widget class="QWidget" name="PageFlipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelTotalCount">
     <property name="text">
      <string>共4条，每页</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboLine" name="lineEditMaxRows">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_plain2">
     <property name="text">
      <string>行</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonFront">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="ComboLine" name="lineEditCurrentPage">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPageCount">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonNext">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboLine</class>
   <extends>QLineEdit</extends>
   <header location="global">src/fend/uicommond/comboline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
